### DISCUSSION TITLE
Can you still use the gnuradio features of the ADI design with this design?

### DISCUSSION THREAD

**Discussion by davidfhorowitz** [open-sdr/openwifi#215] [Q&A] [answered]
Hello, 

I am running your design on a ADI PackRF (ADRV9361-Z7035) and it works great! I am trying to also implement a gnu radio block design using gnu radio companion that I'd like to run when your design is not running. Is this possible with the design as it is now or would it need modifications to the software and/or HDL? Right now I'm having issues getting gnu radio to run. 

Thank you, 

-David

**Comment by JiaoXianjun** [comment, marked as answer]
Normally no. 

Because we have deleted some of the original ADI modules, like DMA stuffs, to save FPGA resources. To make it work, you need to check the original ADI reference design (where we start from), and add the missing block back.

**Reply by davidfhorowitz** [reply]
Got it. Thank you.
